addons/binding/org.openhab.binding.sinope/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="sinope" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="gateway">
+		<label>Sinopé Gateway</label>
+		<description>Sinopé Gateway Configuration</description>
+
+		<config-description>
+			<parameter name="hostname" type="text">
+				<label>Hostname</label>
+				<description>Hostname of the Sinopé Gateway</description>
+				<context>network-address</context>
+				<required>true</required>
+			</parameter>
+			<parameter name="port" type="integer">
+				<label>Port</label>
+				<description>The port that the Sinopé Gateway listens on</description>
+				<default>4550</default>
+			</parameter>
+			<parameter name="gatewayId" type="text">
+				<label>Gateway ID</label>
+				<description>The Sinopé gateway ID (as printed on the back-side, e.g. "xxxx xxxx xxxx xxxx")</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="apiKey" type="text">
+				<label>API Key</label>
+				<description>Use sinope-core application to generate your api key</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="refresh" type="integer" required="false">
+				<label>Refresh Interval</label>
+				<description>The number of seconds between fetches from the Sinopé Gateway.</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="thermostat">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway" />
+		</supported-bridge-type-refs>
+		<label>Sinopé Thermostat</label>
+		<description>Sinopé Thermostat control</description>
+		<channels>
+			<channel id="insideTemperature" typeId="insideTemperature" />
+			<channel id="outsideTemperature" typeId="outsideTemperature" />
+			<channel id="setpointTemperature" typeId="setpointTemperature" />
+			<channel id="setpointMode" typeId="setpointMode" />
+			<channel id="heatingLevel" typeId="heatingLevel" />
+		</channels>
+		<config-description>
+			<parameter name="deviceId" type="text" required="true">
+				<label>Thermostat device ID</label>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="insideTemperature">
+		<item-type>Number</item-type>
+		<label>Temperature</label>
+		<description>Current inside temperature in degrees celcius</description>
+		<category>Temperature</category>
+		<!-- As specified by manufacturer -->
+		<state min="-10" max="70" step="0.01" pattern="%.2f °C" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="outsideTemperature">
+		<item-type>Number</item-type>
+		<label>Temperature</label>
+		<description>Current outside temperature in degrees celsius</description>
```

Comment:
Instead of fixing it to celsius, you might want to use the new unit support, see https://www.eclipse.org/smarthome/documentation/development/bindings/thing-definition.html#state-description and https://www.eclipse.org/smarthome/blog/2018/02/22/units-of-measurement.html.